FreeRTOS-Plus/Source/FreeRTOS-Plus-CLI/FreeRTOS_CLI.c

```diff
@@ -241,7 +243,7 @@ const char *pcReturn = NULL;
 		}
 
 		/* Find the start of the next string. */
-		while( ( ( *pcCommandString ) != 0x00 ) && ( ( *pcCommandString ) == ' ' ) )
+		while(( *pcCommandString ) == ' ')

```

Comment:
Looks like this no longer checking for the string terminator, which means it would access beyond the end of the supplied string.

Reply:
pointless check with the && operator.

Reply:
if ( * pcCommandString) ==  '  ' then ( * pcCommandString) != 0x00.
if ( * pcCommandString) !=  '  ' then break while.

Reply:
I agree with @devprodest .